packages/@react-spectrum/list/src/styles.css

```diff
@@ -437,14 +437,29 @@
 
 .react-spectrum-ListView--draggable .react-spectrum-ListViewItem {
   padding-inline-start: 0;
-}
 
-.react-spectrum-ListViewItem--draggable {
-  .react-spectrum-ListViewItem-checkbox input {
-    inset-inline-start: 0;
+  .react-spectrum-ListViewItem-checkbox {
+    input {
+      inset-inline-start: 0;
+    }
   }
 }
 
+.react-spectrum-ListViewItem--dropTarget {
+  background-color: var(--spectrum-alias-highlight-selected);
+  box-shadow:
+    inset 2px 0 0 0 var(--spectrum-table-cell-border-color-key-focus),
+    inset -2px 0 0 0 var(--spectrum-table-cell-border-color-key-focus),
+    inset 0 -3px 0 0 var(--spectrum-table-cell-border-color-key-focus),
```

Comment:
why the -3?

Reply:
1px was being hidden by the bottom border, so this fixed that.